src/code/models/node.coffee

```diff
@@ -5,23 +5,21 @@ tr = require '../utils/translate'
 
 module.exports = class Node extends GraphPrimitive
 
-  constructor: (nodeSpec={x:0,y:0,title:"untitled",image:null}, key) ->
+  constructor: (nodeSpec={x:0,y:0,title:"untitled",image:null,initialValue:50}, key) ->
     super()
     if key
       @key = key
     @links = []
-    @x = nodeSpec.x
-    @y = nodeSpec.y
-    @title = nodeSpec.title
-    @image = nodeSpec.image
-    @color = nodeSpec.color or Colors[0].value
+    {@x, @y, @title, @image, @initialValue} = nodeSpec
```

Comment:
Looks like this is missing the assignment to `@color`?
`{@x, @y, @title, @image, @color, @initialValue} = nodeSpec`

Or perhaps the line below needs to change back to `@color = nodeSpec.color or Colors[0].value`, or if color is never going to be specified in nodeSpec, then just `@color = Colors[0].value`.